rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -100,7 +103,7 @@ pub(crate) fn run(params: client::GuiParams) -> Result<()> {
                 }
             }
         })
-        .setup(|app| {
+        .setup(move |app| {
```

Comment:
why did we need to add `move`?

Reply:
So that `flag_elevated` gets moved in and we can write it to the logs at debug level. The elevated flag is set if we relaunch to elevate, in which case stderr will get disconnected, so I wanted to log it to prove that it works as intended.